src/shaders/_prelude.vertex.glsl

```diff
@@ -32,6 +32,16 @@ vec3 mix_globe_mercator(vec3 globe, vec3 mercator, float t) {
 #endif
 }
 
+#ifdef PROJECTION_GLOBE_VIEW
+vec3 extrudeOnGlobeSurface(vec2 extrude, vec3 pos_normal, vec3 up_dir, float zoom_transition) {
+    vec3 normal = normalize(mix(pos_normal, up_dir, zoom_transition));
```

Comment:
It's tangential and not something to change here, but I wonder if we should use a fast branch when `u_zoom_transition == 0.0` (most common use case) to avoid extra instructions, wdyt?